UnitedIncome/components
# [APP-5244] Add Color & Illuminated Pencil Illustrations

### Description
> Please provide a brief description of your changes below.

This adds the `color` and `illuminated` pencil illustrations to Storybook. These were provided by April.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

1. Launch repo locally
2. Observe the new illustrations

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

New illustrations in action
[![Image from Gyazo](https://i.gyazo.com/29d29d7cb995dbe7e49b954465442b06.gif)](https://gyazo.com/29d29d7cb995dbe7e49b954465442b06)

### Comments
> Any additional comments that will aid with the review process.

I will need these illustrations for the related `client` PR that is soon to follow.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/APP-5244)
